noxfile.py

```diff
@@ -180,3 +181,64 @@ def make(session: nox.Session):
         *(args.make_args or ("clean", "coredocs")),
     ]
     session.run("make", "-C", "docs/docsite", *make_args, external=True)
+
+
+def _rm_tmpdir(tmpdir):
+    rmtree(tmpdir, ignore_errors=True)
+
+
+@nox.session
+def tag(session: nox.Session):
+    """
+    Check the core repo for new releases and create tags in ansible-documentation
+    """
+    install(session, req="tag")
+
+    CORE_REPO = "https://github.com/ansible/ansible"
+    DOCS_REPO = "https://github.com/ansible/ansible-documentation"
+
+    args = list(session.posargs)
+    tmpdir = None
+
+    if not any(arg.startswith(("--core", "--docs")) for arg in args):
+        if not which("git"):
+            session.error("git is not installed or not found int PATH")
+
+        tmpdir = session.create_tmp()
+        core_dir = "".join((str(tmpdir), "/ansible"))
+        docs_dir = "".join((str(tmpdir), "/ansible-docs"))
```

Comment:
`create_tmp()` returns a string (https://nox.thea.codes/en/stable/config.html#nox.sessions.Session.create_tmp), so there is no need to convert it to a string. Also better use `os.path.join` to combine strings to a path:
```suggestion
        core_dir = os.path.join(tmpdir, "ansible")
        docs_dir = os.path.join(tmpdir, "ansible-docs")
```

Reply:
Per other discussions, this has been removed entirely. But thanks for the tip about it already being a string!